th2_data_services/provider/v5/provider_api/grpc.py

```diff
@@ -109,23 +144,46 @@ def search_events(
             limit_for_parent=limit_for_parent,
             metadata_only=metadata_only,
             attached_messages=attached_messages,
+            filters=filters,
         )
         return self.__stub.searchEvents(event_search_request)
 
     def search_messages(
         self,
-        start_timestamp: datetime,
+        start_timestamp: int,
```

Comment:
@Svyat935 
1. Let's add info that start/end timestamps are expected nanoseconds.
2. Add check, that these timestamps have length = 18 else raise an exception